FastSimulation/Calorimetry/test/Makefile

```diff
@@ -0,0 +1,21 @@
+LIBS = $(shell root-config --libs)
```

Comment:
This is not extremely urgent,
but it would be convenient if this would work w/o makefile,
e.g. you the ROOT compilers.